config/config.go

```diff
@@ -81,15 +81,21 @@ type Fmt struct {
 
 // Plan is a terraform plan config
 type Plan struct {
-	Template    string      `yaml:"template"`
-	WhenDestroy WhenDestroy `yaml:"when_destroy,omitempty"`
+	Template      string        `yaml:"template"`
+	WhenDestroy   WhenDestroy   `yaml:"when_destroy,omitempty"`
+	WhenNoChanges WhenNoChanges `yaml:"when_no_changes,omitempty"`
 }
 
 // WhenDestroy is a configuration to notify the plan result contains destroy operation
 type WhenDestroy struct {
 	Template string `yaml:"template"`
 }
 
+// WhenNoChange is a configuration to add a label when the plan result contains no change
+type WhenNoChanges struct {
+	GithubLabel string `yaml:"github_label,omitempty"`
```

Comment:
I think we can say `Label`? Then it can be used for Gitlab as well (see https://docs.gitlab.com/ee/user/project/labels.html)

For slack or others which does not support the label, we can have some validation when reading?

Reply:
Sure, I can update it to `Label`.

I think for this PR, the label implementation on its own is sufficient.